src/components/material/material-buttons/online/MaterialButtonOnlineExternal.tsx

```diff
@@ -44,7 +49,7 @@ const MaterialButtonOnlineExternal: FC<MaterialButtonOnlineExternalProps> = ({
   size,
   trackOnlineView,
   manifestations,
-  dataCy = "material-button-online-external"
+  dataCy = "material-buttons-online-external"
```

Comment:
Why is this changed, btw?
This component is called MaterialButtonOnlineExternal (button in singular)
and the cy attribute is passed along to the Button component (which is also singular/one button.
It is not a biggie i just wondered.
I have tried in the other cases to follow a pattern where the cy attribute is named after the component, in kebab case of course. This would be a deviance of that pattern

Reply:
It is because the dataCy lineage starts in the component called MaterialButtons, and it passes down a string called "material-buttons" which is then appended "-online" to, which is then appended "-external" to in this component. With my change the string is more consistent as it in all circumstances stays the same for the component.

Reply:
Gotcha ;)